pkg/release-controller/release.go

```diff
@@ -68,11 +68,24 @@ func (c imageInfoConfig) GenerateDigestPullSpec() string {
 	return fmt.Sprintf("%s@%s", strings.Split(c.Name, ":")[0], c.Digest)
 }
 
+func (r *Release) HasInconsistencies() bool {
+	for _, tag := range r.Source.Spec.Tags {
+		if _, ok := tag.Annotations[ReleaseAnnotationInconsistency]; ok {
+			return true
+		}
+	}
+	if _, ok := r.Source.ObjectMeta.Annotations[ReleaseAnnotationInconsistency]; ok {
+		return true
+	}
+	return false
+}
+
```

Comment:
This code includes the inconsistencies changes!  Best to keep them separated as the 2 PRs they are.  This way we could accept this PR before the other and vice versa.

Reply:
Fixed:
- the search box is moved to the right side
![Screenshot from 2022-03-07 19-45-51](https://user-images.githubusercontent.com/77680331/157098074-ac1040ed-8ad9-465b-ae25-a906cead1873.png)
- the count of the rows when clicking: 
![Screenshot from 2022-03-07 19-47-08](https://user-images.githubusercontent.com/77680331/157098271-5f1d11bc-f474-4b27-9390-9bfd5094a2f7.png)

Reply:
What can't be fixed (yet):
- searching the collapsed rows. The search box comes from the datatables plugin, I can't find a way to configure it to our needs/setup. 
Possible workaround: write a custom search plugin - investigating.

Reply:
> What can't be fixed (yet):
> 
> * searching the collapsed rows. The search box comes from the datatables plugin, I can't find a way to configure it to our needs/setup.
>   Possible workaround: write a custom search plugin - investigating.

Not sure I understand...
When I first did my test, I specifically searched for a version that was hidden (4.3.12).  It was found, but hidden behind the `click to expand`.  Clicking it showed the release.  The issue there is that any search results shouldnt be hidden.

Reply:
Added a custom search function that works with our setup. 

At the current setup, it is not possible to add the search box in the same line as the header. 
![image](https://user-images.githubusercontent.com/77680331/157109752-361c1b2c-11a6-4792-9b18-084f6fbbccee.png).

I am trying this instead, perhaps it is a bit more pleasing visually: 
![Screenshot from 2022-03-07 21-07-12](https://user-images.githubusercontent.com/77680331/157109882-8e393957-d2f9-4965-99ae-efa1cad6791f.png).

Searching works now: 
![Screenshot from 2022-03-07 21-09-07](https://user-images.githubusercontent.com/77680331/157110251-2610b6f7-48d9-4ceb-a8ec-1e78f961f7ef.png)

Reply:
The size of the search box wasnt the issue.  It just takes up some of the real-estate we are trying to save.  No biggie.  I do think I prefer the smaller search box though.  Seems excessive to be that big and never really utilized at that length.

Reply:
> > What can't be fixed (yet):
> > 
> > * searching the collapsed rows. The search box comes from the datatables plugin, I can't find a way to configure it to our needs/setup.
> >   Possible workaround: write a custom search plugin - investigating.
> 
> Not sure I understand... When I first did my test, I specifically searched for a version that was hidden (4.3.12). It was found, but hidden behind the `click to expand`. Clicking it showed the release. The issue there is that any search results shouldnt be hidden.

If there is a single item returned by the search, clicking on "click to show all" does not extend the table:
![Screenshot from 2022-03-07 21-20-48](https://user-images.githubusercontent.com/77680331/157112228-c59fb4ec-df02-4abb-8e89-84024ffba75e.png)
I haven't been yet able to fix this using the search plugin from DataTables.